kim/kim-ldap/src/main/java/org/kuali/rice/kim/dao/impl/LdapPrincipalDaoImpl.java

```diff
@@ -306,7 +306,7 @@ public Entity getEntityByEmployeeId(String employeeId) {
         }
 
         Map<String, Object> criteria = new HashMap();
-        String key = getLdapAttribute(getKimConstants().getKimLdapIdProperty());
+        String key = getKimConstants().getKimLdapIdProperty();
```

Comment:
I didn't see a commit for KimConstants so I assume the getKimLdapIdProperty() should already be in there?